.github/workflows/release_nightly.yml

```diff
@@ -294,7 +304,8 @@ jobs:
           asset_name: ${{ needs.create-nightly-release.outputs.package_prefix }}-web-extension.zip
           asset_content_type: application/zip
 
-      - name: Upload Firefox extension
+      - name: Upload Firefox extension (signed)
+        if: ${{ steps.sign-firefox.outcome == 'success' }}
         uses: actions/upload-release-asset@v1
```

Comment:
I'm pretty sure the enclosing `${{ }}` are not needed.

Reply:
The docs agree, and I've removed for consistency.

> When you use expressions in an `if` conditional, you may omit the expression syntax (`${{ }}`) because GitHub automatically evaluates the `if` conditional as an expression.